### PR TITLE
[feat] implement NFT smart contract

### PR DESCRIPTION
**Pull request by greatdolphinls**

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -28,6 +29,7 @@ const CONTRACTS = IS_MAINNET
       USDT: '0xde3A24028580884448a5397872046a019649b084'
     }
   } : {
+    GOVERNANCE: '0x914556b16c1220e4af63084dB1acbD4e6f9c65Aa',
```
Comment (bigwampa) [inline config/index.js:32]: Fuji Governance contract is not the same address as Mainnet Governance contract

File: containers/NFTMarketplace/NFTItem/index.js [hunk 2]
```diff
@@ -75,7 +86,7 @@ const NFTItem = ({
         <Grid item xs={12}>
           <ListItem
             title='Minted'
-            value={`${nft.minted || 0} / ${nft.max || 0}`}
+            value={`${nft.supply || 0} / ${nft.max || 0}`}
```
Comment (bigwampa) [inline containers/NFTMarketplace/NFTItem/index.js:89]: seems to repeat the above file?

**Review by bigwampa — APPROVED** [review]

File: config/index.js (hunk 1 above)
Comment (greatdolphinls) [reply config/index.js:32]: I will update it on next PR

File: containers/NFTMarketplace/NFTItem/index.js (hunk 2 above)
Comment (greatdolphinls) [reply containers/NFTMarketplace/NFTItem/index.js:89]: We can see NFT's minted Value on item and detail modal. So this field is repeated.
image

**Pull request merged**